neo-gui/UI/MainForm.cs

```diff
@@ -260,8 +260,12 @@ private void timer1_Tick(object sender, EventArgs e)
             }
             else
             {
-                toolStripProgressBar1.Value = persistence_span.Seconds;
-                toolStripProgressBar1.Style = ProgressBarStyle.Blocks;
+                try
+                {
+                    toolStripProgressBar1.Value = persistence_span.Seconds;
+                    toolStripProgressBar1.Style = ProgressBarStyle.Blocks;
+                }
+                catch { }
```

Comment:
Why will these lines throw exceptions?

Reply:
I you set the value higher than the max or lower than the min, it throws an exception

Reply:
But how can it be higher or lower? We have these lines:

https://github.com/neo-project/neo-gui/blob/3c4ae9a04e5cede1c89313d7da0d4d32eef5deef/neo-gui/UI/MainForm.cs#L255-L260

Reply:
Agree, should be great to have what value he obtains

Reply:
The toolStripProgressBar1's max value was set to 15:
https://github.com/neo-project/neo-gui/blob/master/neo-gui/UI/MainForm.Designer.cs#L558

If I config a private chain with persist block per 20 seconds , persistence_span "16" may enter the "else" clause and cause an exception.

Reply:
Thanks for all!